api/test/unit/config/settings_test.js

```diff
@@ -90,24 +95,28 @@ describe('config setting Test Suite', function() {
         it('Should return false', function() {
           settings.db.maxConnections = null;
           expect(settings.validate().valid).to.equal(false);
+          expect(settings.validate().message).to.equal("maxConnections is required");
```

Comment:
In the test for port , when port is null,the message is "The port must be a number". Here when maxConnections  is null, message is "maxConnections is required". Can have similar messages at both spots.